### DISCUSSION TITLE
Update to Next.js v10

### DISCUSSION THREAD

**Discussion by cktang88** [blitz-js/blitz#1386] [General] [answered]
Will Blitz.js upgrade to v10 relatively soon? Or is the plan to wait a bit for potential bugs to be ironed out?

https://nextjs.org/blog/next-10

**Comment by flybayer** [comment, marked as answer]
@cktang88 already merged! https://github.com/blitz-js/blitz/pull/1385

Will publish a release soon

**Reply by cktang88** [reply]
awesome!
